### ISSUE TITLE
Building application with nxxm takes forever

### ISSUE THREAD

**Issue by piusayowale** [nxxm/nxxm#3]
I tried to build a simple project with nxxm using visual studio, but it is taking too long to finish it keep show configuring.

**Comment by daminetreg** [comment]
The first time configure is called if you selected platform_libs like Boost you get to wait a long time indeed for it to get installed.

May you share the config ? Is that with the getting-started project ?

**Comment by daminetreg** [comment]
Alternatively I would be happy if you share the output of nxxm with the --verbose command line flag.

**Comment by piusayowale** [comment]
> The first time configure is called if you selected platform_libs like Boost you get to wait a long time indeed for it to get installed.
> 
> May you share the config ? Is that with the getting-started project ?

Yes I am trying it with the getting started but I found out the code has some web assembly dep. so I don't think it should work, but I want to wait and see the error at least, unfortunately it is still configuring so I am waiting, but does nxxm rebuild each dependencies for each project? like for instance, I want to believe it is building boost at the moment, maybe that is what it is doing with "configuring" will it go through the building process for any other project that depends on it?

**Comment by daminetreg** [comment]
Indeed the getting started is definitely for webassembly, but there are mentions of Windows builds in the readme from a previous version, we will fix it so that it runs as well on native build with electron Integration in a future release.

Normally all should run fine and actually it will not rebuild it anymore on your machine for this toolchain. We even have the possibility to host the build for the different toolchains and get it just downloaded, though it is not active in the current release, but this will come soon in a minor release before end of the year.

**Comment by piusayowale** [comment]
That will be nice!!!

**Comment by daminetreg** [comment]
Relates to #5
